src/django_upgrade/fixers/request_headers.py

```diff
@@ -58,6 +58,44 @@ def visit_Call(
         )
 
 
+@fixer.register(ast.In)
+def visit_In(
+    state: State,
+    node: ast.In,
+    parents: list[ast.AST],
+) -> Iterable[tuple[Offset, TokenFunc]]:
+    if (
+        type(parents[-1]) is ast.Compare
+        and len(parents[-1].comparators) == 1
```

Comment:
Everything here is checking `parents[-1]` - that's a good sign that this should really be a function that visits `ast.Compare` nodes, *not* `ast.In` nodes